Test/SubscriptionTest.cs

```diff
@@ -305,6 +305,54 @@ public void UpdateSubscription()
             Assert.Equal(newSubscription.NetTermsType, NetTermsType.NET);
         }
 
+        [RecurlyFact(TestEnvironment.Type.Integration)]
+        public void UpdateSubscriptionWithProrationSettings()
+        {
+            var plan = new Plan(GetMockPlanCode(), GetMockPlanName())
+            {
+                Description = "Update Subscription Plan 1"
+            };
+            plan.UnitAmountInCents.Add("USD", 1500);
+            plan.Create();
+            PlansToDeactivateOnDispose.Add(plan);
+
+            var plan2 = new Plan(GetMockPlanCode(), GetMockPlanName())
```

Comment:
Not critical, but you could make this test leaner by _just_ changing `UnitAmountInCents` on the change.

Reply:
I was hoping to see BOTH credits and charges by changing the plan.

Reply:
Ah, fair.